modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -823,90 +823,192 @@ The JSON object is of the form:
-all the raw DICOM data which have been acquired for that visit. It will return an 
-object of the form:
+A GET request to `/candidates/$CandID/$VisitLabel/dicoms` will return a JSON object of
+all the DICOM archives (A.K.A. one DICOM archive per `StudyInstanceUID`) which have been
+acquired for that visit. It will return an object of the form:
 
 ```js
 {
     "Meta" : {
         "CandID" : $CandID,
         "Visit" : $VisitLabel,
     },
-    "DicomTars" : 
-    [
+    "DicomTars" : [
         {
             "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
             "Patientname" : "DCM123_123456_V1",
-            "SeriesInfo" :
-                [{
+            "SeriesInfo" : [
+                {
                     "SeriesDescription" : "MPRAGE_ipat2",
-                    "SeriesNumber" : "2",
+                    "SeriesNumber" : 2,
                     "EchoTime" : "2.98",
                     "RepetitionTime" : "2300",
                     "InversionTime" : "900",
                     "SliceThickness" : "1",
                     "Modality" : "MR",
                     "SeriesUID" : "1.2.3.4.1107",
-                    },
-                    {
+                },
+                {
                     "SeriesDescription" : "BOLD Resting State",
-                    "SeriesNumber" : "5",
+                    "SeriesNumber" : 5,
                     "EchoTime" : "30",
                     "RepetitionTime" : "2100",
                     "InversionTime" : NULL,
                     "SliceThickness" : "3.5",
                     "Modality" : "MR",
                     "SeriesUID" : "3.4.5.6.1507",
-                }]
+                }
+            ]
         },
         {
             "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
             "Patientname" : "DCM456_654321_V1",
-            "SeriesInfo" :
-                [{
-                "SeriesDescription" : "MPRAGE_ipat2",
-                "SeriesNumber" : "2",
-                "EchoTime" : "2.98",
-                "RepetitionTime" : "2300",
-                "InversionTime" : "900",
-                "SliceThickness" : "1",
-                "Modality" : "MR",
-                "SeriesUID" : "1.7.8.9.1296",
-                }]
+            "SeriesInfo" : [
+                {
+                  "SeriesDescription" : "MPRAGE_ipat2",
+                  "SeriesNumber" : 2,
+                  "EchoTime" : "2.98",
+                  "RepetitionTime" : "2300",
+                  "InversionTime" : "900",
+                  "SliceThickness" : "1",
+                  "Modality" : "MR",
+                  "SeriesUID" : "1.7.8.9.1296",
+                }
+            ]
         }
-    ]    
+    ]
 }
 ```
 
-The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET 
+The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET
 scans, respectively.
 
+```
+POST /candidates/$CandID/$VisitLabel/dicoms
```

Comment:
Is there a way to catch the apache error when file size is too big and return an appropriate error? Currently, when the file size exceeds the apache limits, we get the following message:
```
<Response [400]>
{'error': 'There should be one file uploaded per POST request.'}
```
Which is not very informative. At minimum, it should tell the user to check the apache log instead of saying there should be one file uploaded per POST

Reply:
It seems that it is very hard to know whether a file that exceeds the upload size limit was uploaded in PHP. I updated the error message in [this commit](https://github.com/aces/Loris/pull/9154/commits/39d655a9a3e012b0c6160621c8932ac1c7465f37).